packages/block-editor/src/components/block-preview/style.scss

```diff
@@ -9,38 +10,64 @@
 	}
 
 	.block-editor-block-preview__content {
-		padding: $block-padding;
-		border: $border-width solid $light-gray-500;
 		font-family: $editor-font;
 
+		// Vertical alignment.
+		margin-top: 50%;
+
 		> div {
-			transform: scale(0.9);
-			transform-origin: center top;
 			font-family: $editor-font;
 		}
+	}
 
-		> div section {
-			height: auto;
-		}
-
-		> .reusable-block-indicator {
-			display: none;
-		}
+	.block-editor-block-preview__title {
+		margin-bottom: 10px;
+		color: $dark-gray-300;
 	}
 }
 
+// These rules ensure the preview scales smoothly regardless of the container size.
+.editor-block-preview__container {
```

Comment:
I don't think we should be using the `.editor-` namespace. It's inside the `block-editor` package.

Reply:
correct, everything new would start with `.block-editor-block-preivew…`. some old `.editor-…` were kept for backwards compatibility but we will not be adding any new